src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -2542,154 +2460,148 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
         GlobalNames::VerifyUniqueInterObjectName(
             state, state.dataHVACVarRefFlow->VrfUniqueNames, cAlphaArgs(1), cCurrentModuleObject, cAlphaFieldNames(1), ErrorsFound);
 
-        state.dataHVACVarRefFlow->VRF(VRFNum).Name = cAlphaArgs(1);
-        state.dataHVACVarRefFlow->VRF(VRFNum).VRFSystemTypeNum = VRF_HeatPump;
-        state.dataHVACVarRefFlow->VRF(VRFNum).VRFAlgorithmTypeNum = AlgorithmType::FluidTCtrl;
-        state.dataHVACVarRefFlow->VRF(VRFNum).FuelType = "Electricity";
-        state.dataHVACVarRefFlow->VRF(VRFNum).FuelTypeNum = DataGlobalConstants::ResourceType::Electricity;
+        int VRFNum = state.dataHVACVarRefFlow->NumVRFCond_SysCurve + thisNum;
+        auto &thisVrfFluidCtrl = state.dataHVACVarRefFlow->VRF(VRFNum);
+        thisVrfFluidCtrl.Name = cAlphaArgs(1);
+        thisVrfFluidCtrl.VRFSystemTypeNum = VRF_HeatPump;
+        thisVrfFluidCtrl.VRFAlgorithmTypeNum = AlgorithmType::FluidTCtrl;
+        thisVrfFluidCtrl.FuelType = "Electricity";
```

Comment:
Should not need this ultimately. Can always lookup in `ResourceTypeNames` when needed.